cmd/cli/job/describe.go

```diff
@@ -156,6 +188,33 @@ func (o *DescribeOptions) printExecutions(cmd *cobra.Command, executions []*mode
 	return output.Output(cmd, executionCols, tableOptions, executions)
 }
 
+func (o *DescribeOptions) printHistory(cmd *cobra.Command, label string, history []*models.JobHistory) error {
+	if len(history) < 1 {
+		return nil
+	}
+
+	firstTime := history[0].Occurred()
+	timeSinceCol := output.TableColumn[*models.JobHistory]{
+		ColumnConfig: table.ColumnConfig{Name: historyTimeCol.ColumnConfig.Name, WidthMax: 10, WidthMaxEnforcer: text.WrapText},
+		Value:        func(h *models.JobHistory) string { return h.Occurred().Sub(firstTime).String() },
```

Comment:
showing event elapsed time since first even (t0) is interesting and can give quick insight on how long an even took since the job creation. Though it might be confusing with long running jobs, and when we add job updates. Will change this to elapsed since current time and will re-evaluation t0 in the future